test/xUnit/csharp/test_CompletionHelpers.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using Xunit;
+
+namespace PSTests.Parallel
+{
+    public class CompletionHelpersTests
+    {
+        [Theory]
+        [InlineData("", "", "")]
+        [InlineData("\"", "", "\"")]
+        [InlineData("'", "", "'")]
+        [InlineData("\"word\"", "word", "\"")]
+        [InlineData("'word'", "word", "'")]
+        [InlineData("\"word", "word", "\"")]
+        [InlineData("'word", "word", "'")]
+        [InlineData("word\"", "word\"", "")]
+        [InlineData("word'", "word'", "")]
+        [InlineData("\"word's\"", "word's", "\"")]
+        [InlineData("'word\"", "'word\"", "")]
+        [InlineData("\"word'", "\"word'", "")]
```

Comment:
Last two tests look weird. Do we really expect such results? I don't ask to fix here.

Reply:
So to me it makes sense I guess, an empty string is returned when the quotes at start & end don't match. We cannot give a valid single or double quote if this happens.